libraries/Microsoft.Bot.Connector/Authentication/MicrosoftGovernmentAppCredentials.cs

```diff
@@ -74,7 +87,7 @@ public MicrosoftGovernmentAppCredentials(string appId, string password, string t
         /// </value>
         public override string OAuthEndpoint
         {
-            get { return GovernmentAuthenticationConstants.ToChannelFromBotLoginUrl; }
+            get => string.Format(CultureInfo.InvariantCulture, GovernmentAuthenticationConstants.ToChannelFromBotLoginUrlTemplate, ChannelAuthTenant);
```

Comment:
Should modify the MicrosoftGovernmentAppCredentialsTests around this?

Reply:
Ok， I add some UT here. Also improve the UT for MicrosoftAppCredentials